src/utils/slugizeProductArea.ts

```diff
@@ -0,0 +1,15 @@
+export function slugizeProductArea(s) {
+  // Keep this in sync with slugize in ...
+  // https://github.com/getsentry/security-as-code/blob/main/rbac/lib/make-product-owners
```

Comment:
wonder if at a certain point it's possible to use changes in `security-as-code` to add/remove labels since there's a lot of hoops to jump through to add a new product area

Reply:
That would be great! :)